site/content/docs/api-checks/api-checks.md

```diff
@@ -8,16 +8,56 @@ menu:
     identifier: overview-api-checks
 ---
 
-API checks consist of two parts:
+API checks consist of few parts:
 
-**1. An HTTP request** 
+> The following phases are executed under the hood by the Node.js request library and the chai.js assertions library. All checks are executed on Amazon's AWS Lambda infrastructure.
 
-This is where you configure the method, body, headers etc. which determines the response sent back by your API.  
-[Read more about HTTP request settings]({{< ref "request-settings.md" >}})
 
-**2. One or more assertions** 
+**1. Name and tags**
+
+Start as your mean to go on. Pick a sensible name, something that other members of your team are going to understand. If anything goes wrong, a meaningful name will help identify which alert has triggered.
+Tags can relate checks together, they can also determine which checks are shown on your public dashboards.
+[Read more about public dashboards](./dashboards.md)
+![http request](/docs/images/api-checks/overview-name-tag.png)
+
+**2. The HTTP request**
+
+This is where you configure the method, body, headers etc. which determines the response sent back by your API. [Read more about HTTP request settings](request-settings.md)
```

Comment:
@checklyalex I feel this one can also be made more precise in the phrasing. No need to explain what an HTTP request is, but let's go for a bit more of an exhaustive explanation of the base step of the API check.